resources/js/Components/MismatchRow.vue

```diff
@@ -9,7 +9,11 @@
             </wikit-link>
         </td>
         <td :data-header="$i18n('column-wikidata-value')">
-            <wikit-link class="break-line-link" :href="statementUrl" target="_blank">
+            <span class="empty-value" v-if="!mismatch.value_label && !mismatch.wikidata_value">
```

Comment:
We probably want to ensure that `mismatch.wikidata_value` is `null` and test for that here specifically, so that we can differentiate it from a value that happens to be, for example, `0`.

Reply:
it's not null, it's an empty string.